lib/src/http/event_tracker.dart

```diff
@@ -54,6 +54,7 @@ class HttpEventTracker {
         body: null,
         view: _routeTracker.currentRoute,
         order: _timeUtils.timestamp,
+        startedAt: _timeUtils.timestamp,
```

Comment:
The order stores the timestamp already. I think we should use that instead of having another property that stores the same value. 

We could update the property name later on to better reflect what it is.